src/main/java/dk/bec/polonez/reservationsystem/model/Reservation.java

```diff
@@ -33,4 +34,10 @@
     @ManyToOne
     @JoinColumn
     private Offer offer;
+
+    @Override
+    public String toString() {
```

Comment:
"Returns a string representation of the object. In general, the {@code toString} method returns a string that "textually represents" this object. The result should be a concise but informative representation that is easy for a person to read." javadoc dotyczacy metody toString. Mysle, ze ten toString nie jest zgodny z tym opisem. Co wiecej pojawia sie w toString-u jezyk polski, co tez jest bledem.

jak potrzebujecie na bazie reservation stworzyc jakas waidomosc, to trzeba do tego stwrozyc oddzielna klase

Reply:
Zmieniłem trochę metodę i przeniosłem ją do klasy ReservationService pod inną nazwą